pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -329,8 +329,16 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 
 	// Initialize CPU manager
 	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUManager) {
+		var cpuManagerPolicyOptions map[string]string
+		if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUManagerPolicyOptions) {
+			cpuManagerPolicyOptions = nodeConfig.ExperimentalCPUManagerPolicyOptions
+		} else if nodeConfig.ExperimentalCPUManagerPolicyOptions != nil {
+			klog.InfoS("CPU Manager policy options feature gate not enabled, ignoring values", "options", nodeConfig.ExperimentalCPUManagerPolicyOptions)
```

Comment:
shouldn't setting values in the config for a disabled feature be caught at kubelet instantiation time and prevented?

Reply:
field doc just references the `CPUManager` feature, not CPUManagerPolicyOptions

Reply:
field doc: my fault, fixing.
re: kubelet instantation time: I can move the check earlier indeed. On it.